content/sensu-go/5.19/reference/backend.md

```diff
@@ -1243,12 +1244,15 @@ event-log-file: "/var/log/sensu/events.log"{{< /highlight >}}
 ### Log rotation
 
 Event logging supports log rotation via the _SIGHUP_ signal.
-First, rename (move) the current log file.
+To manually rotate event logs, first rename (move) the current log file.
 Then, send the _SIGHUP_ signal to the sensu-backend process so it creates a new log file and starts logging to it.
 
-Here are some log rotate sample configurations:
+Because event log files can grow very quickly for larger Sensu installations, we recommend using `logrotate` to automatically rotate log files.
```

Comment:
Most linux distributions include `logrotate` as a standard utility, but by default it is configured to run once a day. As a result, folks who want to use our example logrotate configuration to rotate logs more frequently than once a day may need to take steps to configure logrotate to run once an hour [as described here](https://unix.stackexchange.com/questions/29574/how-can-i-set-up-logrotate-to-rotate-logs-hourly).